content-test/components/Spotlight.story.js

```diff
@@ -57,6 +57,11 @@ storiesOf("Highlight List", module)
     <Container>
       <Spotlight length={HIGHLIGHTS_LENGTH} page="SPOTLIGHT_STORYBOOK" sites={fakeSpotlightItems.slice(1)} />
     </Container>
+  )
+  .add("placeholder view (data still loading)", () =>
```

Comment:
I'm glad to hear it; feedback on stuff like this is always helpful.  I actually find it an easier way to develop for some things.